plugin/gentoo-common.vim

```diff
@@ -73,7 +73,6 @@ fun! GentooGetPythonTargets()
                 let l:m = l:py->matchstr("^python3.*")->matchstr("\\d*$")
                 if !empty(l:m)
                     eval l:py3s->add(l:m)
-                    continue
```

Comment:
Initially, it was for purpose, but I already forgot the reason.
IIRC it's an artefact of old PRE-merged revisions where there were another branch of code.
And now it is no need for it anymore.

Although, it would anyway be good to test if function would work as expected even on edge-cases after removing this.